README.md

```diff
@@ -63,32 +63,18 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vet_Clinic] <a name="about-project"></a>
 
 > Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[Vet_clinic]** In this project, you will use a relational database to create the initial data structure for a vet clinic. You will create a table to store animals' information, insert some data into it, and query it.
 
-## 🛠 Built With <a name="built-with"></a>
+## 🛠 Built With <a name="built-with">PostgreSQL</a>
 
-### Tech Stack <a name="tech-stack"></a>
+### Tech Stack <a name="tech-stack">PostgreSQL</a>
 
```

Comment:
- [ ] Good job changing the project name and description in the readme. However, As requested by the previous reviewer, most of the other readme sections are untouched. Kindly edit **each section** of the readme (Key Features, Authors, Future Features, ...etc) to better suit your project and make it more professional.